src/PalindromeQuality.cpp

```diff
@@ -12,19 +11,20 @@ using std::pow;
 #include <boost/accumulators/statistics.hpp>
 
 using namespace boost::accumulators;
+using namespace shasta;
 
-typedef accumulator_set<float, features<tag::count, tag::mean, tag::variance>> stats_accumulator;
+using stats_accumulator = accumulator_set<float, features<tag::count, tag::mean, tag::variance>>;
 
 
 double qualityCharToErrorProbability(char q) {
     return pow(10, double(q - 33) / -10.0);
 }
 
 
-bool shasta::classifyPalindromicQScores(span<char> qualities){
-    double relativeMeanDifference = 0.09;
-    double minimumMean = 0.15;
-    double minimumVariance = 0.025;
+bool shasta::isPalindromic(span<char> qualities,
```

Comment:
Please format this consistently with other Shasta code. The continuation line should have only one extra tab compared to the preceding line.